src/js/profile/wearable/widget/wearable/Slider.js

```diff
@@ -527,6 +527,7 @@
 				} else {
 					return CoreSliderPrototype._setValue.call(self, value);
```

Comment:
src/js/core/widget/core/Slider.js does not return anything. I think this return is useless. Please consider removing it instead of adding return at the end of the function.

Reply:
I based on js doc above method. There is {number|null}

Reply:
I am not sure if this function returns number. I let you make the decision about this.

Reply:
I'm afraid that method `CoreSliderPrototype._setValue` could returns number. For backward compatibility with others version I prefer left this "return" statement